code/modules/events/sentience.dm

```diff
@@ -4,46 +4,46 @@
 	INVOKE_ASYNC(src, .proc/make_sentient_mob)
 
 /datum/event/sentience/proc/make_sentient_mob()
-	var/list/candidates = SSghost_spawns.poll_candidates("Do you want to awaken as a sentient being?", ROLE_SENTIENT, TRUE)
 	var/list/potential = list()
-	var/sentience_type = SENTIENCE_ORGANIC
 
 	for(var/mob/living/simple_animal/L in GLOB.alive_mob_list)
 		var/turf/T = get_turf(L)
-		if (!is_station_level(T.z))
+		if(!is_station_level(T.z))
 			continue
-		if(!(L in GLOB.player_list) && !L.mind && (L.sentience_type == sentience_type))
+		if(!(L in GLOB.player_list) && !L.mind && (L.sentience_type == SENTIENCE_ORGANIC))
 			potential += L
 
-	if(!candidates.len || !potential.len) //if there are no players or simple animals to choose from, then end
+	if(!length(potential)) // If there are somehow no simple animals to choose from, then end.
 		return FALSE
-
 	var/mob/living/simple_animal/SA = pick(potential)
-	var/mob/SG = pick(candidates)
-
-	var/sentience_report = "<font size=3><b>[command_name()] Medium-Priority Update</b></font>"
-
-	var/data = pick("scans from our long-range sensors", "our sophisticated probabilistic models", "our omnipotence", "the communications traffic on your station", "energy emissions we detected", "\[REDACTED\]", "Steve")
-	var/pets = pick("animals", "pets", "simple animals", "lesser lifeforms", "\[REDACTED\]")
-	var/strength = pick("human", "skrell", "vox", "grey", "diona", "IPC", "tajaran", "vulpakanin", "kidan", "plasmaman", "drask",
-					 "slime", "monkey", "moderate", "lizard", "security", "command", "clown", "mime", "low", "very low", "greytide", "catgirl", "\[REDACTED\]")
-
-	sentience_report += "<br><br>Based on [data], we believe that one of the station's [pets] has developed [strength] level intelligence, and the ability to communicate."
 
+	var/list/candidates = SSghost_spawns.poll_candidates("Do you want to awaken as \a [SA]?", ROLE_SENTIENT, TRUE, source = SA)
+	if(!length(candidates) || QDELETED(SA)) // If there's no candidates or the mob got deleted.
+		return
 
+	var/mob/SG = pick(candidates)
 
 	SA.key = SG.key
-	SA.universal_speak = 1
+	SA.universal_speak = TRUE
 	SA.sentience_act()
-	SA.can_collar = 1
+	SA.can_collar = TRUE
 	SA.maxHealth = max(SA.maxHealth, 200)
 	SA.health = SA.maxHealth
 	SA.del_on_death = FALSE
 	greet_sentient(SA)
+
+	var/sentience_report = "<font size=3><b>[command_name()] Medium-Priority Update</b></font>"
+
+	var/data = pick("scans from our long-range sensors", "our sophisticated probabilistic models", "our omnipotence", "the communications traffic on your station", "energy emissions we detected", "\[REDACTED]", "Steve")
```

Comment:
Isn't the `[REDACTED]` now going to display as `[REDACTED`?

Reply:
Tested with Gearbot and it seems fine

Reply:
Yeah, you only need to escape the opening bracket. It makes more sense maybe with both being escaped, but VSCode's syntax highlighting doesn't like it:
![image](https://user-images.githubusercontent.com/57483089/126962874-5380083a-aab7-4b9c-910a-96e6b8f5ddd3.png)